shared/chat/conversation/attachment-input/index.js

```diff
@@ -37,14 +38,23 @@ class RenderAttachmentInput extends Component<Props, State> {
   }
 
   render() {
+    const input = this.props.inputs[this.state.index]
     const count = this.props.inputs.length
-    const currentTitle =
-      (this.props.inputs[this.state.index] && this.props.inputs[this.state.index].title) || ''
+    const currentTitle = (input && input.title) || ''
+    const currentPath = (input && input.filename) || ''
+    const isImage = !!input && input.type === 'Image'
 
     return (
-      <PopupDialog onClose={this.props.onClose}>
+      <PopupDialog onClose={this.props.onClose} styleContainer={isIPhoneX ? {marginTop: 30} : undefined}>
```

Comment:
this `styleContainer` is an ad-hoc fix for the dialog going up under the status bar on iPhone X. I can remove it from this PR if we want to save it for a generalized solution.